app/javascript/tests/mocks/mockHistories.js

```diff
@@ -0,0 +1,25 @@
+
+const mockHistory1 = {
+  id: 1,
```

Comment:
I would add `was_edited: true` to one of these and test the edited tag functionality